### DISCUSSION TITLE
Github puanı hesaba yansıma sorunu

### DISCUSSION THREAD

**Discussion by ZiyaGungor** [IBAX-io/go-ibax#631] [Soru Cevap - Türkçe]
Telegram kanalında ki videoda gösterildiği gibi yapılmasına rağmen github puanı yansımadı.

**Comment by Yusufknk** [comment]
Github puanlarının gelip gelmediğini kontrol etmek için My dashboard-Retweet-Earned&Redeemed bölümünden kontrol edin. Puan yansımamış ise tekrardan deneyin.
